config/environments/development.rb

```diff
@@ -33,17 +44,23 @@
   config.eager_load = false
 
   require 'rack-webauth/test'
-  config.middleware.use(
-    Rack::Webauth::Test,
-    user: Settings.WEBAUTH.USER,
-    mail: Settings.WEBAUTH.MAIL,
-    ldapprivgroup: Settings.WEBAUTH.LDAPPRIVGROUP,
-    suaffiliation: Settings.WEBAUTH.SUAFFILIATION,
-    displayname: Settings.WEBAUTH.DISPLAYNAME,
-    ldapauthrule: Settings.WEBAUTH.LDAPAUTHRULE
-  )
+  if Settings.WEBAUTH
```

Comment:
hmm, not sure i understand the suggestion... for argo-dev, we don't need config values for, e.g., Settings.WEBAUTH.USER, just something to keep the config from invoking config.middleware.use(Rack::Webauth::Test, ...).  i.e. the only instances with config values for things like Settings.WEBAUTH.USER are laptops.

if we can't nil out Settings.WEBAUTH, can you think of a cleaner approach than something like setting Settings.WEBAUTH.USER to empty and checking for emptiness?

i guess another approach would be to have Settings.WEBAUTH default to nil in the root settings.yml, and have laptop configs set things like USER.